app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp_command, current_file, future_file = command.split()
```

Comment:
Your code will broke in next cases:
command = 
"cp"
"cp file1.txt"
"cp file1.txt file2.txt file3.txt"